### PR TITLE
Add an option to exit program.

### PR DESCRIPTION
**Pull request by doug-myers2**
This adds an option to the top level menu to cleanly exit the program.

### CODE REVIEW

File: Jukebox.py [hunk 1]
```diff
@@ -107,6 +108,8 @@ def play(self):
     elif cmd == "3":
         jukebox.next()
         time.sleep(2)
+    elif cmd == "4":
```
Comment (doug-myers) [inline Jukebox.py]: Maybe it should be option 0.

**Review by doug-myers — CHANGES_REQUESTED** [review]
I think it would be better to be option 0 since I don't want it to be in the middle of other options when other options are added.
Also move the code up.

File: Jukebox.py (hunk 1 above)
Comment (doug-myers2) [reply Jukebox.py]: I agree.

Comment (doug-myers2) [reply Jukebox.py]: I made the necessary changes

Comment (doug-myers2) [reply Jukebox.py]: I have made the necessary changes.

**Review by doug-myers — APPROVED** [review]
Looks good.

**Pull request merged**